ui/src/app/components/admin/job-templates/job-template-show/job-template-show.component.html

```diff
@@ -0,0 +1,142 @@
+                [partValidation]="jobTemplateFormEntry.formPart.partValidation">
+              </app-string-part>
+              <app-string-sequence-part
+                *ngSwitchCase="'set-field'"
+                [isShow]="isShow"
+                [name]="jobTemplateFormEntry.formPart.name"
+                [valueChanges]="changes"
+                [value]="jobTemplateFormEntry.value"
+                [property]="jobTemplateFormEntry.formPart.property"
+                [partValidation]="jobTemplateFormEntry.formPart.partValidation">
+              </app-string-sequence-part>
+              <app-key-string-value-part
+                *ngSwitchCase="'key-value-field'"
+                [isShow]="isShow"
+                [name]="jobTemplateFormEntry.formPart.name"
+                [valueChanges]="changes"
+                [value]="jobTemplateFormEntry.value"
+                [property]="jobTemplateFormEntry.formPart.property"
+                [partValidation]="jobTemplateFormEntry.formPart.partValidation">
+              </app-key-string-value-part>
+              <app-guid-part
+                *ngSwitchCase="'guid-field'"
+                [isShow]="isShow"
+                [name]="jobTemplateFormEntry.formPart.name"
+                [valueChanges]="changes"
+                [value]="jobTemplateFormEntry.value"
+                [property]="jobTemplateFormEntry.formPart.property"
+                [partValidation]="jobTemplateFormEntry.formPart.partValidation">
+              </app-guid-part>
+              <app-cron-quartz-part
+                *ngSwitchCase="'cron-quartz-field'"
+                [isShow]="isShow"
+                [name]="jobTemplateFormEntry.formPart.name"
+                [valueChanges]="changes"
+                [value]="jobTemplateFormEntry.value"
+                [property]="jobTemplateFormEntry.formPart.property">
+              </app-cron-quartz-part>
+              <app-select-part
+                *ngSwitchCase="'select-field'"
+                [isShow]="isShow"
+                [name]="jobTemplateFormEntry.formPart.name"
+                [valueChanges]="changes"
+                [value]="jobTemplateFormEntry.value"
+                [property]="jobTemplateFormEntry.formPart.property"
+                [options]="jobTemplateFormEntry.formPart.options"
+                [partValidation]="jobTemplateFormEntry.formPart.partValidation">
+              </app-select-part>
+              <app-boolean-part
+                *ngSwitchCase="'boolean-field'"
+                [isShow]="isShow"
+                [name]="jobTemplateFormEntry.formPart.name"
+                [valueChanges]="changes"
+                [value]="jobTemplateFormEntry.value"
+                [property]="jobTemplateFormEntry.formPart.property">
+              </app-boolean-part>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </form>
+</div>
+
+<h3 *ngIf="!loading && !isSuccessfullyLoaded">
```

Comment:
Is it not possible to cover this case with a toast?

Reply:
fixed